tool/tsh/common/help.go

```diff
@@ -40,10 +40,10 @@ EXAMPLES:
   For passwordless authentication use:
   $ tsh login --auth=passwordless`
 
-	// missingPrincipalsFooter is printed at the bottom of `tsh ls` when no results are returned.
-	missingPrincipalsFooter = `
-  Not seeing nodes? Your user may be missing Linux principals. If trying Teleport for the first time, follow this guide:
+	// emptyNodesFooter is printed at the bottom of `tsh ls` when no results are returned for nodes.
+	emptyNodesFooter = `
+  Not seeing nodes? Your user's roles may not have matching node labels. If trying Teleport for the first time, follow this guide:
 
-https://goteleport.com/docs/getting-started/linux-server/#step-46-create-a-teleport-user-and-set-up-two-factor-authentication
+https://goteleport.com/docs/server-access/getting-started
```

Comment:
+1

Reply:
I tried. Didn't work

Reply:
@rosstimothy Id say we go forward with this then unless you had any other ideas.

Reply:
The server access calls out `tsh` more directly and is focused on adding server access, not hosting and then services.  That's why I changed it.

Reply:
Would it be possible to include the relevant information within the `tsh ls` output rather than link to the docs? That way:
- We don't need to keep track of drift between the docs and `tsh`
- Users don't need to switch contexts and follow a getting started guide that may not apply to them (e.g., if there's a misconfiguration they need to talk to a Teleport admin at their org about, or they were never intended to access servers in the first place without, say, an Access Request)

Reply:
The two most likely reasons are that there are no nodes that match the listing (`tsh ls`, `tsh ls env=test`) or the user's roles do not allow them to see that listing.  We could put that wording in.  thoughts?

Reply:
Sounds good to me!

Reply:
@rosstimothy please see update.  Ex

```bash
Node Name Address Labels
--------- ------- ------


  Not seeing nodes? Either no nodes are available or roles for your user do not match to node labels.

  Check with your Teleport cluster administrator that your user's roles should have nodes available.
```